firstvoices/backend/models/dictionary.py

```diff
@@ -136,6 +137,12 @@ class DictionaryEntry(AudienceMixin, RelatedMediaMixin, BaseControlledSiteConten
         models.CharField(max_length=DEFAULT_TITLE_LENGTH), blank=True, default=list
     )
 
+    import_job = models.ForeignKey(
+        ImportJob,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
Open to suggestions for the value on `on_delete`. 
My decision to choose `SET_NULL` was so that in future if we allow ImportJob instance(s) to be deleted from DB, we would still like to keep the entries.
Another alternative can be to set this to a specific value using `on_delete=SET(..)`, e.g. string `IMPORT_JOB_DELETED` or something to track if the related job was deleted.